index.html

```diff
@@ -144,6 +145,7 @@ <h2 class="stories1"> Website Protfolio </h2>
               </div>
           </div>            
         </div>
+  </div>
      </section>
      <br><br><br><br><br>
                                <!--ABOUT ME SECTION-->
```

Comment:
- [ ] The background image under your `about me` section is not properly positioned and that has created unwanted space to the right of the section. Kindly look into it and fix.

### Your Design
[your-about-me.webm](https://user-images.githubusercontent.com/26597432/206869994-ddf8143c-b560-43b5-99d7-f6cc7e8f9931.webm)

### Figma Design
![figma-about-me](https://user-images.githubusercontent.com/26597432/206870007-5153194a-aa93-4723-af65-1063e0bb7876.png)